libraries/appbase/application.cpp

```diff
@@ -18,104 +18,140 @@ using bpo::options_description;
 using bpo::variables_map;
 using std::cout;
 
+namespace{
+// TODO: delete after HF2
+/**
+ * @brief Fix which renames default data directory and also moves config into subdirectory configs, so users do not need to do it themselves
+ *        Applies fixes only when users did not specify some custom paths to data-dir and config file
+ *
+ * @param actual_data_dir
+ * @param cfg_data_dir
+ * @param cfg_config
+ */
+/*void fix_deprecated_data_folder_structure(const bfs::path& actual_data_dir,
```

Comment:
Why is this code is disabled?

Reply:
As mentioned on standup, the config file migration is part of the missing polishing...